metal_header/template_device.h

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2018 SiFive, Inc */
+/* SPDX-License-Identifier: Apache-2.0 */
+
```

Comment:
I assume this template is for developer to use by cloning the file and edit/add the contents as needed.
Can you add a brief comment on top of the file